package.json

```diff
@@ -1,14 +1,14 @@
 {
   "devDependencies": {
-    "@babel/core": "7.0.0-beta.52",
-    "@babel/node": "7.0.0-beta.52",
-    "@babel/plugin-proposal-class-properties": "7.0.0-beta.52",
-    "@babel/plugin-proposal-optional-chaining": "7.0.0-beta.52",
-    "@babel/plugin-transform-runtime": "7.0.0-beta.52",
-    "@babel/preset-env": "7.0.0-beta.52",
-    "@babel/preset-flow": "7.0.0-beta.52",
-    "@babel/preset-react": "7.0.0-beta.52",
-    "@babel/runtime": "7.0.0-beta.52",
+    "@babel/core": "7.0.0",
```

Comment:
we should add back the `^` on the versions here since we are out of beta's and there will be no breaking changes between v7 versions

Reply:
👍